desktop/components/email/client/editorial_signup.coffee

```diff
@@ -59,7 +59,6 @@ module.exports = class EditorialSignupView extends Backbone.View
     else
       mediator.trigger('open:auth', {
         mode: 'register',
-        redirectTo: window.location.href,
```

Comment:
Similar to above, @kanaabe it looks like this was maybe unintentional? This skips onboarding and just keeps you on the current page. Now, you'll go thru onboarding, and get redirected after onboarding back to the original page.